darc/darc.go

```diff
@@ -293,6 +293,20 @@ func (r Rules) Copy() Rules {
 	return rCopy
 }
 
+// IsSubset checks whether the receiver is a subset of the given actions set.
```

Comment:
Seems like this code should be unit tested. Is there a reason you didn't/couldn't?

Reply:
added a unit test for this one too